### PR TITLE
adds constable outfit in cargocrate

### PR DESCRIPTION
**Pull request by Tlaltecuhtli**
:cl:
add: contraband crate with british police outfit
/:cl:
**why is it good for gaming:**
now they can buy thematic items and sec can do fun rp

**description of what is added:**
items added in the security vendor:
an hat with same properties of the helmet 
a suit with same properties of normal sec suit for 
a whistle with 10s cooldown
a pair of white gloves 
a conversion kit which can turn a stun baton in a crappier police baton (no free batons and no "upgraded versions*") 
for 1100 credits

prices are so sec can buy the whole reskin kit within 20 minutes of the round 

tl note: police baton is more garbage than the roundstart free sec baton its for thematic rp

a

### CODE REVIEW

**Comment by trollbreeder** [comment]
"Oi mate you got that insuls loicense?"

**Comment by Tharcoonvagh** [comment]
"Oh bollocks some wanker stole it"

**Comment by Iatots** [comment]
orange visibility vest + armor vest = yellow armored high visibility vest

Also shouldn't the sleeves be white? I thought the kit was white shirt + black jacket.

**Comment by Tlaltecuhtli** [comment]
> orange visibility vest + armor vest = yellow armored high visibility vest
> 
> Also shouldn't the sleeves be white? I thought the kit was white shirt + black jacket.

image its all black

**Comment by Anonmare** [comment]
Make sure the whistle can be worn in the neck slot, like the camera, when not in use

**Comment by Bawhoppen** [comment]
Why are you saying Zimbabwean dollars? is this a joke I don't understand
Isn't there hyper-inflation in Zimbabwe where it costs like 200 billion for a loaf of bread?

**Comment by Bawhoppen** [comment]
also shouldn't the clothes be in the clothing vendor?

File: code/modules/clothing/masks/hailer.dm [hunk 1]
```diff
@@ -176,6 +176,23 @@ GLOBAL_LIST_INIT(hailer_phrases, list(
...
+	item_state = "whistle"
+	custom_price = 150
+	actions_types = list(/datum/action/item_action/halt)
+
+/obj/item/clothing/mask/whistle/ui_action_click(mob/user, action)
+	whistle()
+
+/obj/item/clothing/mask/whistle/verb/whistle()
+	if(cooldown < world.time - 100)
+		usr.audible_message("<font color='red' size='5'><b>HALT!</b></font>")
```
Comment (optimumtact) [inline code/modules/clothing/masks/hailer.dm]: this is cringe, why is it even a verb

**Comment by Ohagi-Chan** [comment]
"Police baton"
TRUNCHEON PLEASE

File: code/game/objects/items/stunbaton.dm [hunk 2]
```diff
@@ -22,6 +22,8 @@
 	var/throw_hit_chance = 35
 	var/preload_cell_type //if not empty the baton starts with this type of cell
```
Comment (bobbahbrown) [inline code/game/objects/items/stunbaton.dm]: why the empty line

Comment (Tlaltecuhtli) [reply code/game/objects/items/stunbaton.dm]: space is deleted

**Comment by optimumtact** [comment]
Yeah, this can be some kind of space ruin, or a cargo crate, or perhaps even a rare spawn in security randomly.

But there's absolutely zero justification for it to be in the sec tech vendor.

Figure out another way of getting it in the round please.

**Comment by cacogen** [comment]
did you make the sprites?

**Comment by Tlaltecuhtli** [comment]
@optimumtact request adressed?

**Pull request merged**